packages/fxa-auth-server/lib/firestore-db.ts

```diff
@@ -15,8 +15,10 @@ export function setupFirestore(config: ConfigType) {
     delete fsConfig.credentials;
   }
 
+  const testing = !(fsConfig.keyFilename || fsConfig.credentials);
+
   // Utilize the local firestore emulator when the env indicates
-  if (process.env.FIRESTORE_EMULATOR_HOST) {
+  if (process.env.FIRESTORE_EMULATOR_HOST || testing) {
```

Comment:
hmm we don't actually use the value of `FIRESTORE_EMULATOR_HOST` 🙃

Reply:
Yep, that's true, the Google SDK does though, and for our code to work right with it we need to configure firestore slightly differently like this.

Reply:
Oh that's right I keep forgetting the Google SDK is the reason we use a env var directly in the first place.  Thanks for the reminder.